wordle-web/pages/game.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+    <v-row>
+        <v-col>
+            <v-card>
+                <h1>Wordles with Turtles</h1>
+                <p> Yertle was a turtle who knew oh so many wordles. As a young turtle, he was a bit of a nerdle (he even wore a girtle!). Now he's a bit of a flirtle, though his looks make milk curdle, and his favorite scent is myrtle and his favorite color is purple. He calls his home a yurtle!</p>
+                <p> Yertle was a turtle who knew oh so many wordles. As a young turtle, he was a bit of a nerdle (he even wore a girtle!). Now he's a bit of a flirtle, though his looks make milk curdle, and his favorite scent is myrtle and his favorite color is purple. He calls his home a yurtle!</p>
+                <p> Yertle was a turtle who knew oh so many wordles. As a young turtle, he was a bit of a nerdle (he even wore a girtle!). Now he's a bit of a flirtle, though his looks make milk curdle, and his favorite scent is myrtle and his favorite color is purple. He calls his home a yurtle!</p>
+            </v-card>
+            <v-card-actions>
+                <v-btn color="primary" nuxt to="/"> Back to Main </v-btn>
+            </v-card-actions>
```

Comment:
This section seems to be missing some <v-card-text> tags to help with the padding issues on the card. Would also recommend using <the v-card-title> for the header.

Reply:
Thank you. I'm adjusting for the next assignment.